poetry.lock

```diff
@@ -655,39 +729,37 @@ files = [
 
 [[package]]
 name = "pandas"
-version = "1.5.3"
+version = "2.0.1"
```

Comment:
Note this PR will be blocked untill we can verify there are no breaking changes with pandas 2.0.0

Reply:
Looks like the problem was with a newer version of `SQLAlchemy = 2.0` which was a dependency of `pandasql`. After adding a dependency on SQLAlchemy 1.4, the error is fixed without upgrading the version for `pandas`. We do not need to update the version of `pandas` anymore.

With 
- pandas 1.5.3
- SQLAlchemy - 2.0
- pandasql - 0.7.3 
I was getting the following exception

```
sqlalchemy.exc.ObjectNotExecutableError: Not an executable object: 
```

which is fixed now with 
- pandas 1.5.3
- SQLAlchemy - 1.4
- pandasql - 0.7.3